### PR TITLE
X flow id guidance

### LINKED ISSUE #251: X-Flow-ID Guidelines
The new guideline(s) should include:
- How to handle `X-Flow-ID`
- What are the boundaries (scope) of the `X-Flow-ID`
- Which properties should apply to `X-Flow-ID`
- When to generate a `X-Flow-ID` and how should it look like
- Further explanations and reasoning

Related to / triggered by #228.

### PR DESCRIPTION
**Pull request by gewald**
fixes #251.

### CODE REVIEW

File: chapters/proprietary-headers.adoc [hunk 1]
```diff
@@ -121,3 +120,53 @@ entity in a subsequent request. In such cases, the proprietary headers
...
 
+[#233]
+== {MUST} Use `X-Flow-ID`
+
+The *Flow-Id* is a generic parameter to be through service APIs and events,
+that is should be written into log files and traces. A Consequent usage of the
+_Flow-Id_ facilitates the tracking of call flows through our system and allows
+the correlation of service activities initiated by a specific call. This is
+extremely helpful for operational troubleshooting and log analysis.
+
```
Comment (tfrauenstein) [inline chapters/proprietary-headers.adoc]: add: main use case of flow-id is to track service calls of our SaaS fashion commerce platform and initiated internal processing flows (executed synchronously via APIs or asynchronously via published events).

**Comment by whiskeysierra** [comment]
:+1:

File: chapters/proprietary-headers.adoc (hunk 1 above)
Comment (tfrauenstein) [reply chapters/proprietary-headers.adoc]: @gewald Thank you for this PR -- well done!

Comment (gewald) [reply chapters/proprietary-headers.adoc]: @tfrauenstein done

**Comment by tfrauenstein** [comment]
👍

**Pull request merged**